lib/winpki/windows_test.go

```diff
@@ -38,111 +33,6 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-// TestGenerateCredentials verifies that the smartcard certificates generated
-// by Teleport meet the requirements for Windows logon.
-func TestGenerateCredentials(t *testing.T) {
```

Comment:
Why was this test removed? I think there's some important logic covered by this one.

Reply:
This test is the same as the one in `lib/srv/desktop/windows_server_test.go` and here it causes circular dependencies.